src/epcpm/staticmodbustoc.py

```diff
@@ -0,0 +1,136 @@
+import attr
+
+import epcpm.staticmodbusmodel
+import epyqlib.utils.general
+
+
+builders = epyqlib.utils.general.TypeMap()
+
+
+def export(c_path, h_path, staticmodbus_model, skip_sunspec):
+    builder = builders.wrap(
+        wrapped=staticmodbus_model.root,
+        parameter_uuid_finder=staticmodbus_model.node_from_uuid,
+        c_path=c_path,
+        h_path=h_path,
+        skip_sunspec=skip_sunspec,
+    )
+
+    c_path.parent.mkdir(parents=True, exist_ok=True)
+    builder.gen()
+
+
+def gen(self, first=0):
+    lines = []
+
+    for member in self.wrapped.children[first:]:
+        builder = builders.wrap(
+            wrapped=member,
+            parameter_uuid_finder=self.parameter_uuid_finder,
+            c_path=self.c_path,
+            h_path=self.h_path,
+            skip_sunspec=self.skip_sunspec,
+        )
+        lines.extend(builder.gen())
+        lines.append("")
+
+    return lines
+
+
```

Comment:
I think this is unused?

```suggestion
```

Reply:
Inadvertent copy paste. Addressed in ad2366f22522e1df6811fd02213d118243cb054c.